WalletWasabi.Gui/Config.cs

```diff
@@ -19,118 +20,134 @@ namespace WalletWasabi.Gui
 	[JsonObject(MemberSerialization.OptIn)]
 	public class Config : IConfig
 	{
+		public const int DefaultPrivacyLevelSome = 2;
+		public const int DefaultPrivacyLevelFine = 21;
+		public const int DefaultPrivacyLevelStrong = 50;
+		public const int DefaultMixUntilAnonymitySet = 50;
+		public const int DefaultTorSock5Port = 9050;
+		public static readonly Money DefaultDustThreshold = Money.Coins(0.0001m);
 		/// <inheritdoc />
 		public string FilePath { get; private set; }
 
 		[JsonProperty(PropertyName = "Network")]
 		[JsonConverter(typeof(NetworkJsonConverter))]
-		public Network Network { get; internal set; }
+		public Network Network { get; internal set; } = Network.Main;
 
-		[JsonProperty(PropertyName = "MainNetBackendUriV3")]
+		[DefaultValue("http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion/")]
+		[JsonProperty(PropertyName = "MainNetBackendUriV3", DefaultValueHandling = DefaultValueHandling.Populate)]
 		public string MainNetBackendUriV3 { get; private set; }
 
-		[JsonProperty(PropertyName = "TestNetBackendUriV3")]
+		[DefaultValue("http://testwnp3fugjln6vh5vpj7mvq3lkqqwjj3c2aafyu7laxz42kgwh2rad.onion/")]
+		[JsonProperty(PropertyName = "TestNetBackendUriV3", DefaultValueHandling = DefaultValueHandling.Populate)]
 		public string TestNetBackendUriV3 { get; private set; }
 
-		[JsonProperty(PropertyName = "MainNetFallbackBackendUri")]
+		[DefaultValue("https://wasabiwallet.io/")]
+		[JsonProperty(PropertyName = "MainNetFallbackBackendUri", DefaultValueHandling = DefaultValueHandling.Populate)]
 		public string MainNetFallbackBackendUri { get; private set; }
 
-		[JsonProperty(PropertyName = "TestNetFallbackBackendUri")]
+		[DefaultValue("https://wasabiwallet.co/")]
+		[JsonProperty(PropertyName = "TestNetFallbackBackendUri", DefaultValueHandling = DefaultValueHandling.Populate)]
 		public string TestNetFallbackBackendUri { get; private set; }
 
-		[JsonProperty(PropertyName = "RegTestBackendUriV3")]
+		[DefaultValue("http://localhost:37127/")]
+		[JsonProperty(PropertyName = "RegTestBackendUriV3", DefaultValueHandling = DefaultValueHandling.Populate)]
 		public string RegTestBackendUriV3 { get; private set; }
 
-		[JsonProperty(PropertyName = "UseTor")]
-		public bool? UseTor { get; internal set; }
+		[DefaultValue(true)]
+		[JsonProperty(PropertyName = "UseTor", DefaultValueHandling = DefaultValueHandling.Populate)]
+		public bool UseTor { get; internal set; }
 
-		[JsonProperty(PropertyName = "TorSocks5EndPoint")]
-		[JsonConverter(typeof(EndPointJsonConverter), Constants.DefaultTorSocksPort)]
-		public EndPoint TorSocks5EndPoint { get; internal set; }
+        [JsonProperty(PropertyName = "TorSocks5EndPoint")]
```

Comment:
No important but there are spaces here. Could you remove them, please?

Reply:
shit why my settings sometimes revert... I polluted many project with space without knowing...